adminSiteServer/apiRouter.ts

```diff
@@ -3295,21 +3311,8 @@ const createPatchConfigAndFullConfigForChartView = async (
     return { patchConfig: patchConfigToSave, fullConfig }
 }
 
-interface ApiChartViewOverview {
-    id: number
-    slug: string
-    parent: {
-        id: number
-        title: string
-    }
-    updatedAt: string | null
-    lastEditedByUser: string | null
-    chartConfigId: string
-    title: string
-}
-
 getRouteWithROTransaction(apiRouter, "/chartViews", async (req, res, trx) => {
-    type ChartViewRow = Pick<DbPlainChartView, "id" | "slug" | "updatedAt"> & {
+    type ChartViewRow = Pick<DbPlainChartView, "id" | "name" | "updatedAt"> & {
```

Comment:
Same as below - you select more than the type describes. Better to update the type definition.